packages/DraftEditor/src/text-block-parser/index.tsx

```diff
@@ -0,0 +1,81 @@
+import { convertFromRaw, EditorState, RawDraftContentState } from "draft-js";
+import { stateToHTML } from "draft-js-export-html";
+import parseHTML, { domToReact } from "html-react-parser";
+import React from "react";
+import {
+  ANCHOR_TAG,
+  BR_TAG,
+  FIGURE_TAG,
+  HORIZONTAL_RULE,
+  HR_TAG,
+  INPUT_TAG,
+  P_TAG,
+  SUBMIT,
+} from "../constants";
+import { IReactHtmlParserArgs, TComponent } from "../interfaces";
+import isValidJSON from "./is-valid-json";
+
+// tslint:disable-next-line: no-var-requires
+const entities = require("entities");
+
+function ReactHtmlParser(html: string, LinkWrapper?: TComponent) {
+  const options = {
+    replace: ({ attribs, children, name: tagName }: IReactHtmlParserArgs) => {
+      if (tagName === FIGURE_TAG) {
```

Comment:
instead use `if`, maybe would be better using `switch`, up to you.

Reply:
I think not because in some cases it is a condition
Doc
https://www.npmjs.com/package/html-react-parser